packages/react-native/scripts/codegen/__tests__/generate-specs-cli-executor-test.js

```diff
@@ -83,6 +66,8 @@ describe('generateSpec', () => {
       libraryType,
     );
 
-    expect(mkdirpSyncInvoked).toBe(1);
+    // Verifying that mkdirSync was called once with the correct arguments
```

Comment:
nit: Remove self describing comment.
```suggestion
```

Reply:
Removed the self describing comment @huntie